lib/matplotlib/mpl-data/fonts/ttf/local.conf

```diff
@@ -1,32 +0,0 @@
-<?xml version="1.0"?>
-<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
-<!-- /etc/fonts.conf file to configure system font access -->
-<fontconfig>
-        <!--  Enable sub-pixel rendering
-        <match target="font">
-                <test qual="all" name="rgba">
-                        <const>unknown</const>
-                </test>
-                <edit name="rgba" mode="assign"><const>rgb</const></edit>
-        </match>
-         -->
-
-        <alias>
-                <family>serif</family>
-                <prefer>
-                        <family>Bitstream Vera Serif</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>sans-serif</family>
-                <prefer>
-                        <family>Bitstream Vera Sans</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>monospace</family>
-                <prefer>
-                        <family>Bitstream Vera Sans Mono</family>
-                </prefer>
-        </alias>
-</fontconfig>
```

Comment:
I actually didn't realize this was here.  I think this should be updated to use DejaVu rather than just deleted.  It sets the default font types if a user is opting for the font-config mode rather than using matplotlib's own font lookup system.

It's not strictly necessary, but removing this will certainly change behavior for those that are relying on it.